onboarding/apps/bookmarks/templates/bookmarks/base.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  {% load static %}
+  <meta charset="UTF-8">
+  <link rel="stylesheet" type="text/css" href="{% static 'accounts/bootstrap.css' %}">
+  <link href="{% static 'bookmarks/css/all.css' %}" rel="stylesheet">
+  <link href="{% static 'bookmarks/css/css.css' %}" rel="stylesheet">
+  {% block title %}
+  {% endblock %}
+</head>
+<body>
+  <nav class="navbar navbar-light bg-light justify-content-evenly">
+  <a class="navbar-brand">Bookmarks</a>
+  <form class="form-inline" method="GET" style="align-items: unset" id="filter_form">
```

Comment:
Use https://django-filter.readthedocs.io/en/master/guide/usage.html#the-template